docs/features.md

```diff
@@ -110,27 +110,26 @@ Note that in this mode, only pending pods with a running `chaos-handler` init co
 
 ## Notifier
 
-When creating a disruption, you may wish to be alerted of important lifecycle warnings (disruption found no target, chaos pod is stuck on removal, target is failing, target is recovering, etc.) through the Notifier module of the chaos-controller. On each occurrence, these events will be propagated through the different set up notifiers (currently `noop/console`, `slack` and `datadog` are implemented).
+When creating a disruption or a disruption cron, you may wish to be alerted of important lifecycle warnings (e.g., disruption found no target, chaos pod is stuck on removal, target is failing, target is recovering, etc.) through the Notifier module of the chaos-controller. On each occurrence, these events will be propagated through the different set up notifiers (currently `noop/console`, `slack`, and `datadog` are implemented).
 
-You can find the complete list of the events sent out by the controller [here](/api/v1beta1/events.go#L24).
+You can find the complete list of the events sent out by the controller [here](/api/v1beta1/events.go#L97).
 
 Any setup/config error will be logged at controller startup.
 
 ### Slack
 
-The `slack` notifier requires a slack API Token to connect to your org's slack workspace.
```

Comment:
Why remove the mention that we need an api token?

Reply:
Oups, mistake 🙈